tests/common/block_utils.rs

```diff
@@ -287,8 +266,11 @@ pub fn os_hints(
         block_hash: Default::default(),
     };
 
+    // Convert the Blockifier storage into an OS-compatible one
+    let contract_storage_map = build_starknet_storage(&mut blockifier_state);
+
     let execution_helper = ExecutionHelperWrapper::new(
-        state,
+        contract_storage_map,
```

Comment:
Now, it seems like we have two similar contract storage related structures: `contract_storage_map` in `execution_helper` and `contracts` in  `os_input`. Do we need both or is it just temporary?

Reply:
I see the same behaviour in the Python VM structures so I'd say this is fine, at least for now.